release.nix

```diff
@@ -27,7 +27,7 @@ let
           [ curl bison flex perl libxml2 libxslt bzip2 xz
             pkgconfig sqlite libsodium boehmgc
             docbook5 docbook5_xsl
-            autoconf-archive
+            autoconf-archive libyamlcpp
```

Comment:
This should be conditional on `stdenv.isDarwin` since we don't need libyamlcpp elsewhere.